lang/cs/Org.Apache.REEF.Bridge/Org.Apache.REEF.Bridge.vcxproj

```diff
@@ -41,6 +41,7 @@ under the License.
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <Keyword>ManagedCProj</Keyword>
     <RootNamespace>Org.Apache.REEF.Bridge</RootNamespace>
+    <WindowsTargetPlatformVersion Condition="'$(VisualStudioVersion)' >= '15.0'">10.0.14393.0</WindowsTargetPlatformVersion>
```

Comment:
It appears that only having VS2017 installed forces you to build for windows 10, but the KB mentioned above will give backwards compatibility.